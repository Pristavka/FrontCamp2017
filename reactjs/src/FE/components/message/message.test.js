import React from 'react';
import Message from './massage';

it('Should render a Message', () => {
  const wrapper = shallow(<Message />);
  expect(wrapper).toMatchSnapshot();
});

