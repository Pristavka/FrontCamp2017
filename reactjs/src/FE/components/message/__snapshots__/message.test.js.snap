// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Message 1`] = `
<div
  className="successMsg"
>
  Your post has added successfuly!
</div>
`;
